gulpfile.js

```diff
@@ -101,7 +101,7 @@ function watch() {
     gulp.watch(
         ['tokens/*.yml'],
         gulp.series([
-            'positioning-formats',
+            // 'positioning-formats', // AssertionError [ERR_ASSERTION]: Task never defined: positioning-formats - SeanMcP
```

Comment:
Does this work in master? It is my only hesitation with approving the PR

Reply:
I don't think it matters... I think this is like, you could define some positioning tokens and then you could pump them out in different formats, but we never have used it, so no big deal. Polaris used a spacing format to turn https://github.com/Shopify/polaris-tokens/blob/master/tokens/spacing.yml into https://github.com/Shopify/polaris-tokens/blob/master/dist/spacing.map.scss with https://github.com/Shopify/polaris-tokens/blob/master/formats/spacing-map.scss.js

Reply:
I am not able to checkout this branch and run it. I"m not sure why.

Reply:
Weird I wonder what your issue is @spencerkingman-niche :thinking:.

@SeanMcP I just checked this repo out, ran `npm ci` with node `14.15.0`, and got the same error you saw in this branch:
```
[09:58:36] 'watch' errored after 1.18 ms
[09:58:36] AssertionError [ERR_ASSERTION]: Task never defined: positioning-formats
    at getFunction (/home/brendan/code/expedite-tokens/node_modules/undertaker/lib/helpers/normalizeArgs.js:15:5)
```

So really this is actually out of scope of this PR. One last suggestion is you could delete the line instead of commenting it out, but I'm going to approve nonetheless.

Reply:
I couldn't check out the branch because it was created under a different user "SeanMcP" than ("NicheInc")

Reply:
@spencerkingman-niche Sorry about that, I edited the file initially on GitHub, which automatically forked the repo into my personal account. In the future, I'll make sure to do it locally.